sig-aws/README.md

```diff
@@ -11,7 +11,7 @@ To understand how this file is generated, see generator/README.md.
 Covers maintaining, supporting, and using Kubernetes hosted on AWS Cloud.
 
 ## Meetings
-* [Fridays at 17:00 UTC](https://zoom.us/my/k8ssigaws) (biweekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=17:00&tz=UTC).
+* On [Zoom](https://zoom.us/my/k8ssigaws), Fridays at 9:00 Pacific, biweekly. [Convert to your timezone](http://www.thetimezoneconverter.com/?t=9:00&tz=PT).
```

Comment:
This file is generated by running `make`. We should modify what we want in `sigs.yaml` and then run `make`. But seems like the Makefile doesn't support to use `PT` now. The docker image which `make` uses should be modified, IMO.

BTW, there is an issue on timezone concern: #841 

cc @grodrigues3 @cblecker

Reply:
Thanks - for now I just set it to 16:00 UTC along with a comment linking #841.